setup.cfg

```diff
@@ -26,6 +26,7 @@ ignore_errors = True
 [tool:pytest]
 addopts = --black --flake8 --mypy --ignore-glob=*/version.py
 flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503
+asyncio_mode = auto
```

Comment:
I am curious if you tried "strict". I did with narrativelog and the tests still passed.